index.html

```diff
@@ -239,22 +238,33 @@ <h1>There are many advantages to taking an API-first development approach. And w
           '#%RAML 1.0',
           'title : Hello World # required title\n',
           '/greeting: # optional resource',
-          '\tget: # HTTP method declaration',
-          '\t\tresponse: # Declare a response',
```

Comment:
This had an incorrect value of "response" when it should have been a "responseS". Also conversion RAML->OAS didn't work properly with tabs so I switched them to double whitespaces/